app/models/manageiq/providers/embedded_ansible/automation_manager/configuration_script_source.rb

```diff
@@ -102,6 +102,11 @@ def sync_playbooks
 
       configuration_script_payloads.reload
     end
+    true
+  end
+
+  def sync_queue(auth_user = nil)
```

Comment:
https://github.com/ManageIQ/manageiq-ui-classic/pull/5849#issuecomment-513940527 should have commented here directly